README.md

```diff
@@ -24,6 +36,23 @@ cd virtualtabletop
 npm install                                                                             # uses the Node.js package manager to install all dependencies
 ```
 
```

Comment:
Since you're adding Windows, here's one for mac:

```suggestion

### MacOS
Using [brew](https://brew.sh/):

    brew install node
    git clone --recurse-submodules https://github.com/ArnoldSmith86/virtualtabletop.git
    cd virtualtabletop
    npm install

```